idunn/blocks/wikipedia.py

```diff
@@ -122,13 +134,13 @@ def wrapper(*args, **kwargs):
             except RequestException:
                 logging.error("Got Request exception in {}".format(f.__name__), exc_info=True)
             except TooManyRequests:
+                cls._prom_tracker.limiter_exception()
                 logging.warning("Got TooManyRequests{}".format(f.__name__), exc_info=True)
             except RedisConnectionError:
                 logging.warning("Got redis ConnectionError{}".format(f.__name__), exc_info=True)
             except TimeoutError:
```

Comment:
I think we need ton counter for each of these exception non ? (maybe just a handler on a generic exception metric ?)

Reply:
done